### ISSUE TITLE
`juliaup info` crashes on MacOS Ventura 13.0

### ISSUE THREAD

**Issue by camilogarciabotero** [JuliaLang/juliaup#487]
Hi,

I followed the instructions to use `juliaup 1.8.10` , then change to the releasepreview channel `juliaup self channel releasepreview`, and that worked fine, but then when executing the `juliaup info` got the following problem:

```
name = 'Juliaup'
operating_system = 'unix:OSX'
crate_version = '1.8.10'
explanation = '''
Panic occurred in file '/Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/ureq-2.5.0/src/rtls.rs' at line 74
'''
cause = 'Failed to add native certificate too root store: BadDer'
method = 'Panic'
backtrace = '''

   0: 0x1082a6a85 - core::result::unwrap_failed::ha988429942445917
   1: 0x108226069 - core::ops::function::FnOnce::call_once::ha0d6f48b9a614fb5
   2: 0x108121612 - once_cell::imp::initialize_or_wait::hca53ff2183dd744c
   3: 0x1082b17df - once_cell::imp::OnceCell<T>::initialize::h3f1d155e6e2aef41
   4: 0x1082231f2 - ureq::agent::AgentBuilder::new::hb1a69231fb9fd917
   5: 0x1080fae98 - juliaup::operations::get_ureq_agent::h8ca3e86b253070b5
   6: 0x108106657 - juliaup::operations::download_juliaup_version::h1a7e6db0c6685e7c
   7: 0x1080109b0 - juliaup::main::hf9faef8b0b0035cc
   8: 0x107ff3916 - std::sys_common::backtrace::__rust_begin_short_backtrace::h2ccf0636b6322bbe
   9: 0x107ff3a32 - std::rt::lang_start::{{closure}}::hbaf7903ca9db3d7f
  10: 0x108032e02 - _main
  11: 0x7ff8178c4310 - <unresolved>'''
```

I'm working on MacOS Venture 13.0. Let me know if more info is needed.

**Comment by davidanthoff** [comment]
I found https://github.com/pact-foundation/pact-reference/issues/156 and https://github.com/rustls/rustls-native-certs/issues/15 that sound somehow related... Not really clear to me what is going on or how we can fix this, but we probably should...

**Comment by wolthom** [comment]
@davidanthoff If I understand the following discussion correctly, the `reqwests` crate had a similar issue:
https://github.com/seanmonstar/reqwest/pull/1316

I suspect a similar root-cause in `ureq` and that the solution would follow the lines of whatever `reqwests` did. The first step may be opening an issue in the `ureq` repository and pointing to this issue and the `reqwests` PR.

What do you think?

**Comment by davidanthoff** [comment]
Yes, from my (very quick and sloppy) read I came to a similar conclusion, i.e. this probably needs to be fixed in `ureq`. If you have the bandwidth (and want) to engage with the `ureq` folks that would be fantastic :)

**Comment by wolthom** [comment]
I'll set up an issue and give some context. Maybe the author is already aware of this or knows of a quick way to improve this / can adopt the `reqwests` solution.

**Comment by wolthom** [comment]
Good news:
https://github.com/algesten/ureq/pull/564

The ureq maintainers reacted very quickly and already have a PR open. Once the PR is merged, we could update the version bounds and see if that fixes the issue.

**Comment by christiangnrd** [comment]
I just tried `juliaup info` on an M1 Mac mini with juliaup version 1.8.12 and macOS 13.0.1 and there was no crash. Maybe it's fixed?

**Comment by camilogarciabotero** [comment]
Yes, it seems to be fixed on `juliaup v1.8.12` Thank you!
